src/main/java/com/uploadcare/api/Image.java

```diff
@@ -0,0 +1,118 @@
+package com.uploadcare.api;
+
+import com.uploadcare.data.FileData;
+import com.uploadcare.data.ImageInfo;
+
+import java.util.Date;
+
+/**
+ * Created by Egor Petushkov
+ * on 09.06.17.
+ */
+public class Image extends File {
+    public static final int UNKNOW_DPI = -1;
+    public enum Orientation {
```

Comment:
System is pretty simple when you get it.
There are 8 orientations which you can split on 2 groups. The first group is when you take an image and rotate it clockwise. The second group is when you horizontally flip an image and rotate it clockwise. Illustration: http://www.impulseadventure.com/photo/images/orient_flag.gif
So once you understand how it works, these names makes sense.